### PR TITLE
refresh from https://github.com/carpentries/styles

### PR DESCRIPTION
**Pull request by mr-c**
refresh from https://github.com/carpentries/styles/commit/3efe66951f3f8bd10d6548c1df766e4e2f34af74

### CODE REVIEW

**Comment by tobyhodges** [comment]
You are updating from the workshop website template and not [the lesson template](https://github.com/carpentries/styles) repo. Is there a reason for this?

**Comment by mr-c** [comment]
Whoops, my mistake! I'll redo this.

File: .github/workflows/website.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          while read -r cmd
+          do
+            eval sudo $cmd
+          done < <(Rscript -e 'cat(remotes::system_requirements("ubuntu", "18.04"), sep = "\n")')
+
+      - run: make site
+      - run: make lesson-check
+        if: always()
+      - run: make lesson-check-all
+        if: always()
```
Comment (tobyhodges) [inline .github/workflows/website.yml:85]: As things stand these steps will raise errors for any code blocks using syntax highlighting for YAML or CWL (if rouge has a highlighting scheme for CWL?). I have opened https://github.com/carpentries/styles/pull/532, which will address that problem.

File: bin/lesson_check.py [hunk 2]
```diff
@@ -79,12 +78,16 @@
...
+    'language-cmake',
+    'language-cpp',
     'language-make',
     'language-matlab',
     'language-python',
     'language-r',
     'language-shell',
-    'language-sql'
+    'language-sql',
+    'warning'
```
Comment (tobyhodges) [inline bin/lesson_check.py:90]: I guess you could add `language-yaml` here for a local fix of the issue mentioned above?

Comment (mr-c) [reply bin/lesson_check.py:90]: I added `--permissive` to the Makefile, so these warnings are no longer fatal. This is ready to go, if you want to approve it :-)

Comment (mr-c) [reply bin/lesson_check.py:90]: And as we have a lot of self-inflicted FIXMEs we have to have `--permissive`

Comment (tobyhodges) [reply bin/lesson_check.py:90]: That works for now, I guess.

**Review by tobyhodges — APPROVED** [review]

**Pull request merged**